cla_public/libs/utils.py

```diff
@@ -28,6 +28,28 @@ def get_locale():
         language_keys
     ) or 'en'
 
+def change_locale(locale):
+    """
+    Set locale cookie
+    """
+    if locale[:2] not in [l for l, d in current_app.config['LANGUAGES']]:
+        abort(404)
```

Comment:
💭1 - Would returning a 404 be correct if `locale` is a query parameter, or should we ignore an unrecognised locale instead?

💭2 - On that train of thought, should we replace the `/locale/<locale>` path with query parameters across the whole site for a consistent approach?